### PR TITLE
store files of disk instead of leveldb

### PR DESCRIPTION
**Pull request by juliangruber**
cc @mafintosh

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -18,7 +20,10 @@ let key;
 const keyPath = `${appPath}/key.txt`;
 try { key = fs.readFileSync(keyPath) } catch (_) {}
 
-const archive = drive.createArchive(key, { live: true });
+const archive = drive.createArchive(key, {
+  live: true,
+  file: name => raf(`${appPath}/files/${name}`)
```
Comment (mafintosh) [inline app.js]: i think we should put them in the downloads folder (whatever the equivalent is on each os). this points to the electron folder right?

Comment (mafintosh) [inline app.js]: maybe @feross has a module that returns the downloads folder for every os? webtorrent probably does something similar

Comment (juliangruber) [inline app.js]: I'm not sure putting it in the downloads folder is the right idea. When you have limited disk like I do you frequently purge it. On the other hand, that's exactly the right thing if you just use this for experimentation for now - then the data you fetch is short lived anyways.

> this points to the electron folder right?

Yup, that's in `Application Support`.

Hm still `~/Downloads` is probably the best folder. A quick npm search didn't reveal any module for it.

What do others think?

Comment (maxogden) [inline app.js]: +1 ~/Downloads folder for Mac. not sure about x-platform. I think `home-dir` is the module for getting the users home dir. I think the folder should be easy to change eventually but for now Downloads seems like a sensible default

**Comment by juliangruber** [comment]
@mafintosh updated

**Comment by mafintosh** [comment]
@juliangruber looks great! 👍

**Comment by feross** [comment]
We used to use an approach like @juliangruber's module, but some advanced users on Windows complained that they changed the location of their Downloads folder in the OS settings.

To get around that, we just used Electron's [`app.getPath(name)`](https://github.com/electron/electron/blob/master/docs/api/app.md#appgetpathname)

```js
app.getPath('downloads')
```

**Pull request merged**